pom.xml

```diff
@@ -27,6 +27,8 @@
   <properties>
     <jenkins.version>2.150.1</jenkins.version>
     <java.level>8</java.level>
+    <!-- TODO fix once released -->
+    <configuration-as-code.version>1.20-20190604.105601-2</configuration-as-code.version>
```

Comment:
@kuisathaverat wooops, there's actually an incrementals in use here. I don't think we should release in this state.

Reply:
it is a test dependency, I guess if I try with the latest version and the test pass is ok to release changing the dependency to the latest version.

Reply:
Waiting for the release today, most likely. CC: @oleg-nenashev @casz